modules/ROOT/pages/autocorrect.adoc

```diff
@@ -0,0 +1,49 @@
+= Spelling Autocorrect plugin
+:navtitle: Spelling Autocorrect
+:description: Autocorrect common spelling mistakes.
+:keywords: autocorrect, capitalization, spell, spelling, spellcheck
+:pluginname: Spelling Autocorrect
+:plugincode: autocorrect
+:pluginminimumplan: tiertwo
+
+
+include::partial$misc/admon-premium-plugin.adoc[]
+
+The {pluginname} plugin adds capabilties to TinyMCE to autocorrect a specified set of spelling errors or capitalization errors, as they are being typed.
```

Comment:
```suggestion
The {pluginname} plugin adds capabilties to {productname} to autocorrect a specified set of spelling errors or capitalization errors, as they are being typed.
```
We should always use the asciidoc variables for this type of thing. That way if it's renamed later it's very easy to change as it's only in one place.